app/qml/ProjectWizardPage.qml

```diff
@@ -0,0 +1,203 @@
+    height: InputStyle.rowHeightHeader
+    width: projectWizardPanel.width
+    color: InputStyle.clrPanelMain
+    rowHeight: InputStyle.rowHeightHeader
+    titleText: qsTr("Create Project")
+
+    onBack: {
+      projectWizardPanel.back()
+    }
+  }
+
+  Item {
+    height: projectWizardPanel.height - header.height - toolbar.height
+    width: projectWizardPanel.width
+    y: header.height
+
+    ColumnLayout {
+      id: contentLayout
+      spacing: InputStyle.panelSpacing
+      anchors.fill: parent
+      anchors.margins: projectWizardPanel.panelMargin
+
+      Label {
+        height: projectWizardPanel.rowheight
+        width: parent.width
+        Layout.preferredHeight: projectWizardPanel.rowHeight
+        horizontalAlignment: Text.AlignLeft
+        verticalAlignment: Text.AlignVCenter
+        text: qsTr("Project name")
+        color: InputStyle.fontColor
+        font.pixelSize: InputStyle.fontPixelSizeNormal
+      }
+
+      TextField {
+        id: projectNameField
+        width: parent.width
+        height: projectWizardPanel.rowHeight
+        font.pixelSize: InputStyle.fontPixelSizeNormal
+        color: projectWizardPanel.fontColor
+        placeholderText: qsTr("Project name")
+        font.capitalization: Font.MixedCase
+        inputMethodHints: Qt.ImhNoPredictiveText
+        Layout.fillWidth: true
+        Layout.preferredHeight: projectWizardPanel.rowHeight
+
+        background: Rectangle {
+          anchors.fill: parent
+          border.color: projectNameField.activeFocus ? InputStyle.fontColor : InputStyle.panelBackgroundLight
+          border.width: projectNameField.activeFocus ? 2 : 1
+          color: InputStyle.clrPanelMain
+          radius: InputStyle.cornerRadius
+        }
+      }
+
+      Label {
+        id: attributesLabel
+        height: projectWizardPanel.rowheight
+        width: parent.width
+        text: qsTr("Attributes")
+        color: InputStyle.fontColor
+        font.pixelSize: InputStyle.fontPixelSizeNormal
+        Layout.preferredHeight: projectWizardPanel.rowHeight
+        horizontalAlignment: Text.AlignLeft
+        verticalAlignment: Text.AlignVCenter
+      }
+
+      ListView {
+        id: fieldList
+        model: __fieldsModel
+        width: parent.width
+        Layout.fillWidth: true
+        Layout.fillHeight: true
+        clip: true
+
+        onCountChanged: {
+          if (fieldList.visible) {
+            fieldList.positionViewAtEnd()
+          }
+        }
+
+        delegate: FieldRow {
+          height: projectWizardPanel.rowHeight
+          width: contentLayout.width
+          color: projectWizardPanel.fontColor
+          widgetList: projectWizardPanel.supportedTypes
+
+          onRemoveClicked: __fieldsModel.removeField(index)
+        }
+      }
+
+      Row {
+        id: listButtonContainer
+        width: parent.width
+        Layout.preferredHeight: projectWizardPanel.rowHeight * 0.8
+        Layout.fillWidth: true
+        Button {
```

Comment:
Can we center this button on the screen? Or maybe put it inside toolbar? (not sure which one is better)